proposed/2023/sponsor-link.md

```diff
@@ -0,0 +1,164 @@
+
+Maintainers don't work on OSS for money. They don't do it for fame. They do it out of pride in craftsmanship and the responsibility for something they believe in. It takes nerves of steel to work for many years on hundreds of thousands of lines of very complex code, with every line of code you touch being visible to the world, knowing that code is used by banks, weapon systems, smartphones, industry, government, and just about everywhere. With the knowing that you'll be ignored and unappreciated until something goes wrong. 
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+#### .nuspec / .csproj MSBuild property
+
+The `.nuspec` will introduce a new collection element called `<sponsors>` which is a collection of zero or more `<sponsor>` elements specifying the sponsor requirements for the package.
+
+```xml
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata minClientVersion="100.0.0.1">
+        <id>dasdas</id>
+        <version>2.0.0</version>
+        <title />
+        <authors>dsadas</authors>
+        <owners />
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>My package description.</description>
+    </metadata>
+    <sponsors>
+        <sponsor type="Patreon" url="https://www.patreon.com/mypatreonusername" />
+        <sponsor type="GitHub Sponsors" url="https://github.com/sponsors/mygithubusername" />
+        <sponsor type="Tidelift" url="https://tidelift.com/subscription/pkg/mypackagename" />
+    </sponsors>
+</package>
+```
+
+For .NET projects that use the `PackageReference` format which draws inputs from the project file, one can create a new `<ItemGroup>` for their sponsors such as:
+
+```xml
+<ItemGroup>
+    <Sponsors Include="Patreon" Url="https://www.patreon.com/mypatreonusername"/>
+    <Sponsors Include="GitHub Sponsors" Url="https://github.com/sponsors/mygithubusername"/>
+    <Sponsors Include="Tidelift" Url="https://tidelift.com/subscription/pkg/mypackagename"/>
+</ItemGroup>
+```
+
+#### NuGet.org
+
+On NuGet.org there will be a new `Sponsor this package` button which will open up a webpage to the first defined sponsor in the package metadata. Given there can be multiple sponsors, there may be an additional experience where one can click "all sponsors" and un-collapse the other options available.
+
+![NuGet.org Sponsor Experience](../2023/resources/NuGetOrgSponsors.png)
+
+#### Visual Studio
+
+In Visual Studio's NuGet package manager UI, there will be a new link that says `Sponsor` which will open up a webpage to the first defined sponsor in the package metadata.
+
+![Visual Studio Sponsor Experience](../2023/resources/VisualStudioSponsors.png)
+
```

Comment:
Can a section about API implementation be added? Will this be part of the search API or included in individual registrations? (search would be preferred as then getting the sponsorship info is only one API request).

Reply:
We will add it as we work on the technical explanation.